programming/vision-tracking.md

```diff
@@ -0,0 +1,42 @@
+# Basics of Vision Tracking
+Vision Tracking!!! It allows the robot to see. It likes shiny tape. You need a camera, raspberry pi, LED ring, and something to vision track.
+
+
+OpenCV Library!! (Open Source Computer Vision Library)
+It's a thing. Use it.
+
+HSV stands for hue saturation value. Each can range from 0-255.
+
+Start by importing numpy and cv2.
```

Comment:
Can you put backticks (`` ` ``) around numpy and cv2, because they are inline code samples